test/jdk/javax/net/ssl/SSLEngine/CheckStatus.java

```diff
@@ -43,42 +43,40 @@
 
```

Comment:
I think this test needs more enhancements. The test is run with jsse.enableCBCProtection disabled and you updated it run with TLSv1.2. Can you think of better test name/summary depending on the test scenario?